### PR TITLE
Jbaker/testing

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #18 

## Summary:

Add unit tests to meet 90% or greater threshold. Some minor component code cleanup to make sure the tests all passed correctly.

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: test/auro-datepicker.test.js [hunk 1]
```diff
@@ -18,9 +13,160 @@ describe('auro-datepicker', () => {
...
   // });
 
-  it('auro-datepicker custom element is defined', async () => {
+  it('sets the CSS class on auro-datepicker > div element', async () => {
+    const el = await fixture(html`
+      <auro-datepicker class="testClass"></auro-datepicker>
+    `);
+
+    expect(el.className).to.equal('testClass');
+  });
```
Comment (blackfalcon) [inline test/auro-datepicker.test.js:22]: This is a test for putting a class name on the root element?

File: test/auro-datepicker.test.js [hunk 2]
```diff
@@ -1,15 +1,10 @@
...
 describe('auro-datepicker', () => {
-  it('sets the CSS class on auro-datepicker > div element', async () => {
-    const el = await fixture(html`
-      <auro-datepicker class="testClass"></auro-datepicker>
-    `);
-
-    expect(el.className).to.equal('testClass');
-  });
-
   // it('auro-datepicker is accessible', async () => {
```
Comment (blackfalcon) [inline test/auro-datepicker.test.js:8]: What's the plan for addressing this test?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: test/auro-datepicker.test.js (hunk 2 above)
Comment (jason-capsule42) [reply test/auro-datepicker.test.js:8]: Tracked in a different ticket: https://github.com/AlaskaAirlines/auro-datepicker/issues/26

**Review by blackfalcon — DISMISSED** [review]

**Comment by blackfalcon** [comment]
@jason-capsule42 rebase off of main and merge.

**Pull request merged**